wordpress-mobile/WordPress-iOS
# Issues/7272 login email screen

Refs #7272 
This is a first pass at wiring up the new login email view controller.  

To test:
- Confirm the visual design was correctly implemented.
- Confirm the form is disabled until entering something resembling an email address. 
- Confirm the button to enter a site address instead opens the self-hosted form (for now). 
- Confirm that a valid wpcom email address advances to the magic link flow. 
- Confirm that a email address not attached to a wpcom account prompts to double check and retry. 
- Run the WordPressKit tests and confirm the tests for the newly added auth-options endpoint pass. 

Needs review: @nheagy 

Screen shots:

![simulator screen shot jun 14 2017 1 17 34 pm](https://user-images.githubusercontent.com/1435271/27148099-a8bc28e8-5104-11e7-972a-4943d4f898c6.png)
![simulator screen shot jun 14 2017 1 17 44 pm](https://user-images.githubusercontent.com/1435271/27148103-aaadfea6-5104-11e7-80e2-a83c5285d183.png)
![simulator screen shot jun 14 2017 1 17 59 pm](https://user-images.githubusercontent.com/1435271/27148104-ac2a942e-5104-11e7-8243-f4363ec97d55.png)
